### DISCUSSION TITLE
High Virginia demand scenario producing unrealistic results

### DISCUSSION THREAD

**Discussion by wwoolard** [NREL/ReEDS-2.0#176] [Q&A: Analysis]
We are trying to run a scenario with all constraint defaults except an increased demand in Virginia through 2050. To do this we are 
 using the demandscen input in cases.csv. We copied  demand_AEO_2023_reference.csv and replaced the values for VA with the values for VA from demand_AEO_2023_high.csv to simulate the scenario. This produced a result where generation in Virginia decreased, which can't be true. Does anyone have any theory on why this might happen, or how we could simulate increased demand in VA through a different method? We are running reeds2.0 and have limited the model under GSw_Region to the nercr/PJM region.

<img width="466" alt="Screenshot 2024-04-22 at 10 30 21 AM" src="https://github.com/NREL/ReEDS-2.0/assets/167785554/b196ed8d-a8de-4dc3-a26f-8039fd9b637d">

**Comment by wesleyjcole** [comment]
Have you looked at the load data to make sure it is coming into the model correctly?  It is possible that increased demand in VA led to more imports, hence the need to verify that the inputs are coming in correctly before diagnosing why the outputs look off.
